pkg/executables/executables.go

```diff
@@ -87,10 +91,10 @@ func redactCreds(cmd string) string {
 	return cmd
 }
 
-func execute(ctx context.Context, cli string, in []byte, args ...string) (stdout bytes.Buffer, err error) {
+func execute(ctx context.Context, cli string, in []byte, envVars map[string]string, args ...string) (stdout bytes.Buffer, err error) {
```

Comment:
do we need to os.Setenv here?

Reply:
I thought we only called os.Setenv in order to lookup envs to redact, but setting the env was in place before we started redacting so I''m adding in back in the `Run()` method where it was originally